src/pages/MainMenu/MainMenu.js

```diff
@@ -13,5 +13,6 @@ export function Menu () {
     <li><Link to='/browse-happs'>Hosting</Link></li>
     <li><Link to='/earnings'>Earnings</Link></li>
     <li><Link to='settings'>Settings</Link></li>
+    <li><Link to='/my-profile'>Edit Profile</Link></li>
```

Comment:
Should we add profile link to the menu?

Reply:
No the profile edit is only accessed through the profile bubble for now. You can always defer to the wireframes if this stuff is unclear, they'll be increasingly reliable :)